openshift/settings.bc-offline-indexing.sh

```diff
@@ -1,8 +1,8 @@
 # Description: A profile to create an offline indexing environment.
 # Setting for the TheOrgBook-BC environments.
-# Uses the existing `devex-von-tools` environment for builds and images
-export TOOLS="devex-von-tools"
-export PROJECT_NAMESPACE="devex-von-bc-tob"
+# Uses the existing `8ad0ea-tools` environment for builds and images
+export TOOLS="8ad0ea-tools"
```

Comment:
@WadeBarnes should we remove the explicit definition of the tools namespace since now a standard project set is used?